### PR TITLE
Api note

### PR DESCRIPTION
**Pull request by tusharphalke19995**
@atulgupta2024 Pleaes verify the api-note file

### CODE REVIEW

File: api_specifications/note-api-0.1.0.yaml [hunk 1]
```diff
@@ -0,0 +1,256 @@
+openapi: 3.0.1
+info:
+  title: Note And Constitute Element API Documentation
+  description: This draft API specification defines the Note and Constitute element registries.
```
Comment (atulgupta2024) [inline api_specifications/note-api-0.1.0.yaml:4]: split note and constituent elements into two different YAML files

File: api_specifications/note-api-0.1.0.yaml [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+          description: Note search failed. Refer to response body for specifics.
+          content:
+            '*/*':
+              schema:
+                $ref: https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/ErrorRes
+  /element/v1/search:
+    post:
+      tags:
+        - Element
+      summary: Search for constitute elements by case type
```
Comment (atulgupta2024) [inline api_specifications/note-api-0.1.0.yaml:92]: it is constituent and not constitute

File: api_specifications/note-api-0.1.0.yaml [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+              schema:
+                $ref: '#/components/schemas/NoteListResponse'
+        '400':
+          description: Note search failed. Refer to response body for specifics.
+          content:
+            '*/*':
+              schema:
+                $ref: https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/ErrorRes
+  /element/v1/search:
+    post:
```
Comment (atulgupta2024) [inline api_specifications/note-api-0.1.0.yaml:89]: just element in the name will eb confusing.  make this 

/constituentelement/v1/create:
/constituentelement/v1/update:
/constituentelement/v1/search:

File: api_specifications/note-api-0.1.0.yaml [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+          type: string
+        workflow:
+          $ref: https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/Workflow
+        additionalDetails:
+          type: object
+          description: This field is meant to hold any extra information that we want to store for a note.
+        auditDetails:
+          $ref: https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/AuditDetails
+        constituteElements:
+          type: array
```
Comment (atulgupta2024) [inline api_specifications/note-api-0.1.0.yaml:153]: this is not a field inside note. we discussed. make it a separate registry

**Review by atulgupta2024 — CHANGES_REQUESTED** [review]

File: api_specifications/note-api-0.1.0.yaml (hunk 1 above)
Comment (tusharphalke19995) [reply api_specifications/note-api-0.1.0.yaml:4]: ok

File: api_specifications/note-api-0.1.0.yaml (hunk 3 above)
Comment (tusharphalke19995) [reply api_specifications/note-api-0.1.0.yaml:89]: ok

File: api_specifications/note-api-0.1.0.yaml (hunk 4 above)
Comment (tusharphalke19995) [reply api_specifications/note-api-0.1.0.yaml:153]: ok

**Review by atulgupta2024 — APPROVED** [review]
approving API definition so that it is into the folder and easier to review from there on.

**Pull request merged**